internal/core/window.rs

```diff
@@ -60,6 +60,11 @@ pub trait WindowAdapter {
     /// Returns the window API.
     fn window(&self) -> &Window;
 
+    /// Show the window if visible is true, hide otherwise.
+    fn set_visible(&self, _visible: bool) -> Result<(), PlatformError> {
```

Comment:
I'm wondering if PlatformError really make sense here.
None of the variant from the enum actually makes sense.
Do you think we'll add varaint in the future that make sense?
Should we just use `String`?

Now, i understand that this is what we return from the public Window::show and Window::hide, so that's a good argument to keep it as is. But it returns that because the Window::show function may need to initialize the global platform if it is not set.

Reply:
Does this deserve a ChangeLog entry?

Reply:
> None of the variant from the enum actually makes sense.

Yes, that's true. At the moment there's no real error case.

> Do you think we'll add varaint in the future that make sense?

It's possible. I do think that we might extend `PlatformError` in the future with specific errors once we identify them, i.e. converting cases where we currently use `Other()`.

So my preference is to keep it. Otherwise there's a chance we end up with a situation like in `dispatch_event()` where we should have returned an Error but didn't, and now we unwrap() when resizing the renderer fails :(
(which may very well happen)

(Technically speaking for the wasm build `set_visible()` can fail as we do our first rendering there, which in turn may fail - so there's at least one use-case, but rather exotic).

> Does this deserve a ChangeLog entry?

Yes, that makes sense. I'll add that.